templates/CRM/Contribute/Form/Contribution.tpl

```diff
@@ -18,13 +18,16 @@
   {if $contributionMode}
     <div class="help">
       {if $contactId && $payNow}
-        {ts 1=$displayName 2=$contributionMode|upper}Use this form to edit a contribution on behalf of %1. <strong>A
-          %2 transaction will be submitted</strong> using the selected payment processor.{/ts}
+        {ts 1=$displayName}Use this form to edit a contribution on behalf of %1.{/ts}
```

Comment:
multil-line, but also incorrect assumption of concatenation. It would be impossible to translate "LIVE" or "TEST" and it might not make grammatical sense in many languages.